manual/src/threelang/tfgrid/machines.md

```diff
@@ -0,0 +1,63 @@
+# Machines Namespace
+
+- To deploy a number of virtual machines on the same network, use the machines namespace.
+
+## Create Operation
+
+- action name: !!tfgrid.machines.create
```

Comment:
Can we rename to tfgrid.machine.create? And add a parameter network which defines the network it belongs too. Network will replace the parameter model_name.